README.md

```diff
@@ -6,13 +6,100 @@ via a dracut module.
 
 
 ## Kernel command line options for coreos-installer running in the initramfs
+
 * coreos.inst=yes - Instruct the installer to run
 * coreos.inst.install_dev - The block device on the system to install to
 * coreos.inst.image_url - The url of the coreos image to install to this device
 * coreos.inst.ignition_url - The url of the coreos ignition config (optional, enter
   coreos.inst.ignition_url=skip to not load an ignition config)
 
-## Testing out the installer script
+## Using the installer on Fedora or RHEL CoreOS
+
+This installer is incorporated into Fedora CoreOS and RHEL CoreOS.
+There are ISO images that can be downloaded that will allow for an
+install to be performed on bare metal hardware either via ISO install
+or via a PXE install.
+
+#### Grab an ISO image and bare metal image
+
+For Fedora CoreOS you can currently grab an ISO image from the output
+of the current development pipeline located
+[here](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/).
+
+The ISO image can install in either legacy boot (BIOS) mode or in UEFI
+mode. You'll have to make sure you download the appropriate related
+image for the mode you'd like to use. It is a good idea to download
+them before doing the install as the artifacts server that is being
+used currently has very slow download speeds.
+
+For example download:
+
+- [fedora-coreos-29.731.iso](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731.iso)
+
+and one of the following two:
+
+- [fedora-coreos-29.731-metal-bios.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-bios.raw.gz)
+- [fedora-coreos-29.731-metal-uefi.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-uefi.raw.gz)
+
+**NOTE** Those links will quickly become broken because we prune
+         builds. You'll have to find your own links from the 
+         [latest directory](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/).
+
+#### Test an ISO based install
+
+You can test an install on a bare metal machine by burning the ISO to
+disk and booting it or using ISO redirection via a LOM interface.
+Alternatively you can use a VM like so:
+
+```
+virt-install --name cdrom --ram 4500 --vcpus 2 --disk size=20 --accelerate --cdrom /path/to/fedora-coreos-29.731.iso --network default
```

Comment:
My gut says to have a qemu example as well. the libvirt tooling tends to be more "automagical" which is great once you understand it, but hides the details that users reading this documentation might want. Having both is probably best to cover both the "I just want to run it in a vm" case and the "I'm looking at how you run it in a vm to gain better understanding" case.

Reply:
ehh - since it's just an example I'd prefer to not since it might be more confusing than helpful. when we make official docs it might be nice to have an *expandable* clicky option for either virt-install or qemu directly, WDYT?

Reply:
I'm coming from the other direction where using `virt-install` is more confusing that helpful. I'd error on the side of using tools that do as little as possible in docs since it's more clear what's happening under the hood.

Reply:
can you give me the equivalent qemu command and I'll paste it in?

Reply:
```
qemu-system-x86_64 -cpu host -accel kvm -cdrom fedora-coreos-29.731.iso -drive file=drivefile,if=virtio,format=raw -nographic -m 5G
```
should work, but I'm having trouble testing it. Will dig in tomorrow. (install dev would be `vda` not `sda` in this case)

Reply:
Added a #TODO to add a qemu command - I want to push this out so I can link people to it and close out some other tickets so let's add the qemu command in a follow up.